### ISSUE TITLE
Display day of the week alongside the page title if it's a date

### ISSUE THREAD

**Issue by Stvad** [roam-unofficial/roam-toolkit#42]

**Comment by ryanguill** [comment]
I think I could do this relatively easily.  To make sure im clear about what the goal is, you want to take a title like:
`May 28th, 2020`
and make it
`Thursday May 28th, 2020`
is that right?

I suppose we could make the option more like "custom formatting of date titles" and have a setting that allows you to provide a mask - if the title is a date we format it using the mask.  Then you could turn it into any format you like.

**Comment by Stvad** [comment]
Yep, your understanding is correct and custom formatting sound great!  
I actually now have a page for each day of the week, with corresponding days referenced to hack this 😛
